src/triage/experiments/multicore.py

```diff
@@ -12,9 +12,13 @@
 class MultiCoreExperiment(ExperimentBase):
     def __init__(self, n_processes=1, n_db_processes=1, *args, **kwargs):
         super(MultiCoreExperiment, self).__init__(*args, **kwargs)
+        if n_processes < 1:
+            raise ValueError('n_processes must be 1 or greater')
+        if n_db_processes < 1:
+            raise ValueError('n_db_processes must be 1 or greater')
```

Comment:
Cool.

Not really sure this is worthwhile, but we could also log a warning or something if these are not set usefully:

    if n_db_processes == 1 and n_processes == 1:
        LOG.warning('this is not recommended dummy')

(I can absolutely see why someone would want to do it, so it should be allowed – _e.g._ if you want to test your configuration and/or investigate bugs. But outside of those cases it seems to me like a bad idea.)